website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,192 @@
+
+#### Footer
+
+By default, the footer displays below the list and contains content and/or buttons (depending on the variant) that provide additional context (or functions) to the list options.
+
+##### Single
+
+The footer shows a total count, which is customizable to fit the context.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-single-variant-footer-with-counter.png)
+
+##### Multiple
+
+In the multiple select variant, the `afterOptions` and `footer` display buttons to expedite the deselection process and optionally a counter (selected / total).
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-multiple-variant-footer.png)
+
+“Show selected” will change the list to display only selected list items while toggling the label of the button to “Show all.”
+
+![Flow showing the behavior of the show selected button and how it toggles the list to only display selected list items](/assets/components/form/super-select/super-select-flow-select-all.png)
+
+If “Show selected” is used when no selections have been made, a message will appear to let the user know.
+
+![Flow showing the behavior of the show selected button when no selections have been made yet.](/assets/components/form/super-select/super-select-flow-no-selections-select-all.png)
+
+Additionally, “Clear selected” will wipe out the selections and revert the Super Select to its deselected state. This button is hidden if there are no selections.
+
+![Flow showing the behavior of the clear selected button and how it reverts the Super Select's state to the default one](/assets/components/form/super-select/super-select-flow-clear-selected.png)
+
+##### Footer with custom content
+
+In either of the two variants, you are able to fill the footer with custom content. This is helpful when wanting to mimic the footer but still customize its contents.
+
+![Showing the footer with custom content inside.](/assets/components/form/super-select/super-select-single-multiple-footer-custom-content.png =836x*) 
+
+#### CustomContent
+
+If enabled, custom content will occupy this space.
+
+![Showing the afterOptions slot with custom content](/assets/components/form/super-select/super-select-after-options-custom-content.png =836x*)
+
+## List
+
+### Positioning
+
+Lists can be positioned to the left, center or right of the trigger, and above or below the trigger to fit more appropriately within the UI. These options are only available when `matchTriggerWidth` is set to `false`.
+
+![Showing the different list positions (left, center and right + top and bottom)](/assets/components/form/super-select/super-select-list-positioning.png =836x*)
+
+### Width
+
+#### Match the trigger
```

Comment:
You _might_ be able to trim out this headline and just say:

```markdown
### Width

By default, the width...
```
Since this is the default experience.